sdk/highlight-ruby/highlight/test/highlight_test.rb

```diff
@@ -2,6 +2,7 @@
 
 class HighlightTest < Minitest::Test
   def test_logger
+
```

Comment:
Made this change to trigger the workflow.
See output: https://github.com/highlight/highlight/actions/runs/5757895336/job/15609618194?pr=6230